net/udp/udp_sendto_unbuffered.c

```diff
@@ -192,6 +192,15 @@ static uint16_t sendto_eventhandler(FAR struct net_driver_s *dev,
 
       else
         {
+          /* Copy the user data into d_appdata and send it */
+
+          devif_send(dev, pstate->st_buffer,
```

Comment:
why change the order

Reply:
New send flow will prepare the d_iob buffer in devif_send(), sendto_ipselect() depends on d_iob to update d_appdata